Bricks/net/tcp/impl/posix.h

```diff
@@ -172,11 +172,35 @@ class SocketHandle : private SocketSystemInitializer {
 #endif
 };
 
+struct IPAndPort {
+  std::string ip;
+  int port;
+  IPAndPort(const std::string& ip = "", int port = 0) : ip(ip), port(port) {}
```

Comment:
Yeah, my bad. Perhaps `= default;` `operator=` for `const IPAndPort` and `IPAndPort&&` ?